src/main/java/lawrence/command/AddTaskCommand.java

```diff
@@ -0,0 +1,72 @@
+package lawrence.command;
+
+import lawrence.database.TaskFileManager;
+import lawrence.parser.InputSource;
+import lawrence.parser.TaskParser;
+import lawrence.task.Task;
+import lawrence.task.TaskList;
+import lawrence.ui.UserInterface;
+import lawrence.utils.DateParser;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
```

Comment:
Wrong import order. Refer to import order as specified in A-CodingStandard. import java.* should come first

Reply:
The import order specified by the [coding standard](https://se-education.org/guides/conventions/java/index.html#package-and-import-statements) only requires that the order of imports is **consistent** across the project, which my project follows. As the original repository does not have any imports in the original `Duke.java` file, there is no import order established prior to my current ordering.

The coding standard's example is merely an example and is not the fixed order that must be followed.

Will keep my current order of imports as it stands, but will modify it accordingly if there is a need to.